### ISSUE TITLE
Typo in Options/General  - "sytem" instead of "system"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#217]
```
There is a typo "sytem" instead of "system" in the first option in 
Options/General. 

It should read "Launch when system starts" instead of "Launch when sytem 
starts".


```

Original issue reported on code.google.com by `krustyjo...@gmail.com` on 20 Apr 2011 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
Thanks
```

Original comment by `boss...@yahoo.fr` on 21 Apr 2011 at 7:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 5 Jun 2011 at 5:30
- Changed state: **NextRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed by IMSDroid 2.0.431
```

Original comment by `boss...@yahoo.fr` on 13 Jul 2011 at 11:52
- Changed state: **Fixed**
